app/javascript/packs/complaint-links.js

```diff
@@ -82,18 +77,23 @@ document.querySelectorAll(".js-close-edit-tta").forEach((el) => {
   })
 })
 
+// Unlink form for TTA and Monitoring
+
 document.querySelectorAll(".js-open-unlink-modal").forEach((el) => {
   el.addEventListener('click', event => {
-    const displayId = event.currentTarget.dataset.displayId
-    // add display id to hidden input form
-    const hiddenInput = document.querySelector("#hidden-tta-display_id")
-    hiddenInput.value = displayId
+    // note: id here refers to monitoring's review id or a TTA issue's display_id
+    const { id, inputField } = event.currentTarget.dataset
+    const hiddenInput = document.querySelector("#hidden-"+inputField)
```

Comment:
[suggestion (nonblocking)] this is totally fine as is, but if you don't know about it, JS now has string interpolation using backticks and dollar signs:

```
`#hidden-${inputField}`
```

Reply:
I'm gonna be honest, I know very little about JS so that is good information for me to know